src/test/groovy/org/web3j/solidity/gradle/plugin/SolidityCompilerTest.groovy

```diff
@@ -34,4 +38,18 @@ class SolidityCompilerTest {
         final def solc = new SolidityCompiler().getExecutable()
         assertThat(solc.name).isEqualTo("solc")
     }
-}
+
+    @Test(expected = RuntimeException.class)
+    void testWindowsDll()
+    {
+        System.setProperty("os.name","Windows 10")
+        final def solc = new SolidityCompiler();
+    }
+
+    @Test
+    void testFakeWindowsDll() {
+        final requiredFile = folder.newFile("vcruntime140.dll")
+        println requiredFile.getParent()
+        assertThat(SolidityCompiler.checkWindowsVcppExists("/var/;/etc/;" + requiredFile.getParent())).isEqualTo(true)
```

Comment:
I think the test should be as before, even reusing the same method eg:

```groovy
@Test
void testFakeWindowsDll() {
    final requiredFile = folder.newFile("vcruntime140.dll")
    testWindowsDll()
}
```

This would also allow to make the utility method `checkWindowsVcppExists` private as is for internal use. 

Also please remove the `println`.